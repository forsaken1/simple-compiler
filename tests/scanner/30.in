+4.3e-5