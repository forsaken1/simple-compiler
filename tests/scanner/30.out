1	1	OPERAT		+
1	2	REAL		4.3e-5